app/models/customer.rb

```diff
@@ -0,0 +1,6 @@
+class Customer < ApplicationRecord
+  enum sector: %w[central_government wider_public_sector]
```

Comment:
Is it worth being explicit with the mappings here?

Reply:
updated.